BaseTools/Tests/TestTools.py

```diff
@@ -45,7 +45,11 @@ def MakeTheTestSuite(localItems):
 
 def GetBaseToolsPaths():
     if sys.platform in ('win32', 'win64'):
-        return [ os.path.join(BaseToolsDir, 'Bin', sys.platform.title()) ]
+        BaseToolsBinPath=os.path.join(BaseToolsDir, 'Bin', sys.platform.title())
+        if(os.path.exists(BaseToolsBinPath)):
+            return [ BaseToolsBinPath ]
+        elif(os.path.exists(BaseToolsBinPath.replace('Win32','Win64'))):
+            return  [ BaseToolsBinPath.replace('Win32','Win64') ]

```

Comment:
why do we care about the host architecture?

Reply:
mingw-w64 gcc make can build two kinds of binaries: 64-bit or 32-bit.